crates/solidity/outputs/cargo/crate/src/extensions/semver/mod.rs

```diff
@@ -0,0 +1,463 @@
+pub mod parser;
+pub mod tests;
```

Comment:
Suggestion: rather than having multiple `#[cfg(test)]` attributes inside `tests/mod.rs`, we can just add that attribute on the module here, so that the entire module won't be included unless we are compiling tests.

```rust
#[cfg(test)]
pub mod tests;
```